src/pages/Dashboard.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
-import Profile from '../components/profile';
+import firebase from '../components/firebase/base';
 
 function Dashboard() {
   return (
     <div className="Dashboard">
       <h1>Dashboard </h1>
       <h4> This is your dashboard </h4>
-      <Profile />
+      {firebase.getCurrentUsername() ? (
+        <h4>{firebase.getCurrentUsername()}</h4>
+      ) : (
+        <h4>Not Logged in</h4>
```

Comment:
Could we have some more consistent capitalization here

Reply:
This soon to be customized - if you meant for the 'Not Logged in' tag. The firebase.getCurrentUsername() is a built-in API